.gitignore

```diff
@@ -30,3 +30,7 @@ go.work.sum
 /main
 dist/
 cache/
+
+# Ignore customized configuration files
+*.toml
+!*.toml.template
```

Comment:
can probably remove this - People primarily consume this package as a binary so it's rare that people will directly try to commit their toml to the repo